README.md

```diff
@@ -34,7 +34,7 @@ An option is a contract giving the buyer _the right, but not the obligation_, to
 
 HegicCallOptions and HegicPutOptions are options contracts. These contracts calculate `fees` (options prices), `create` new options, `exercise` options contracts on behalf of the holders and `exchange` holders' assets using [the Uniswap Protocol](https://github.com/Uniswap) for sending liquidity back to the pool during the process of exercising.
 
-HegicETHPool and HegicERCPool are liquidity pools contracts. These non-custodial contracts accumulate liquidity from providers (writers). Pooled liquidity is used for selling (writing) options contracts to the buyers (holders). After a liquidity provider calls the `provide` method, they send ETH / [ERC-20 tokens](https://eips.ethereum.org/EIPS/eip-20) to the liquidity pool contract and receive writeETH / writeERC (ERC-20) tokens to their address. To leave the pool a liqudity provider calls the `withdraw` method, burns their writeETH / writeERC tokens and receives ETH / ERC-20 tokens to their address.
+HegicETHPool and HegicERCPool are liquidity pools contracts. These non-custodial contracts accumulate liquidity from providers (writers). Pooled liquidity is used for selling (writing) options contracts to the buyers (holders). After a liquidity provider calls the `provide` method, they send ETH / [ERC-20 tokens](https://eips.ethereum.org/EIPS/eip-20) to the liquidity pool contract and receive writeETH / writeERC (ERC-20) tokens to their address. To leave the pool a liquidity provider calls the `withdraw` method, burns their writeETH / writeERC tokens and receives ETH / ERC-20 tokens to their address.
```

Comment:
Since I modified the file from the branch I created the workflow PR this contains the lint fixes in the file.